redis/src/cluster.rs

```diff
@@ -64,6 +64,12 @@ use crate::cluster_routing::{Routable, RoutingInfo, Slot, SLOT_SIZE};
 
 type SlotMap = BTreeMap<u16, [String; 2]>;
 
+#[derive(Clone, Copy)]
```

Comment:
This should stay above its `impl` block. I guess I missed this the first time around?

Reply:
We've removed impl block for TlsMode.